### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/zh/guide/artalk.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#547] [Announcements]
# https://plugin-comment2.vuejs.press/zh/guide/artalk.html

Artalk 是一款简洁的自托管评论系统，你可以在服务器上轻松部署并置入前端页面中。 来到你的博客，或是任意位置，放置 Artalk 评论框，让页面具备丰富的社会化功能。 部署 Artalk 服务端 请参见 Artalk 文档 (https://artalk.js.org/guide/deploy.html)。 配置 请配置 provider: 

https://plugin-comment2.vuejs.press/zh/guide/artalk.html

**Comment by Firstnsnd** [comment]
test

**Reply by Firstnsnd** [reply]
test

**Reply by Firstnsnd** [reply]
test

**Comment by FlyingPig278** [comment]
Test

**Comment by zou88288** [comment]
fg
